x-pack/plugins/security_solution/public/common/components/search_bar/index.tsx

```diff
@@ -304,16 +305,15 @@ export const SearchBarComponent = memo<SiemSearchBarProps & PropsFromRedux>(
       }
     }, [sourcererDataView, fieldFormats]);
 
-    const onTimeRangeChange = useCallback(
-      ({ query, dateRange }) => {
+    const onTimeRangeChange = useCallback<NonNullable<StatefulSearchBarProps['onTimeRangeChange']>>(
+      ({ dateRange }) => {
         const isQuickSelection = dateRange.from.includes('now') || dateRange.to.includes('now');
         updateSearch({
           end: dateRange.to,
           filterManager,
           id,
           isInvalid: false,
           isQuickSelection,
-          query,
```

Comment:
is there a reason to remove `query` property here?
Method is called `onTimeRangeChange`, so ideally it  shouldn't change query as well. But wondering if it removing query can bring any side effects

Reply:
It looks like `onTimeRangeChange` doesn't have `query` param, so if I understand correctly we either way were passing `undefined`
https://github.com/patrykkopycinski/kibana/blob/chore/react-18-callback-any/src/plugins/unified_search/public/search_bar/search_bar.tsx#L92